### PR TITLE
Remove softmax from logits in nasnet_mobile.py

### PR DESCRIPTION
**Pull request by vfdev-5**
Probably, we need to remove softmax from logits

### CODE REVIEW

**Comment by Cadene** [comment]
Thanks @vfdev-5  :)
